src/esp/gfx/RenderCamera.cpp

```diff
@@ -76,6 +76,18 @@ RenderCamera& RenderCamera::setProjectionMatrix(int width,
   return *this;
 }
 
+RenderCamera& RenderCamera::setOrthoProjectionMatrix(int width,
+                                                     int height,
+                                                     float znear,
+                                                     float zfar) {
+  auto orthoMat = Mn::Matrix4::orthographicProjection(Mn::Vector2{16.0f, 9.0f},
+                                                      znear, zfar);
+
+  MagnumCamera::setProjectionMatrix(orthoMat).setViewport(
+      Magnum::Vector2i(width, height));
```

Comment:
The {16.0f, 9.0f} is arbitrary, hacky and inappropriate, but I am still figuring out the appropriate configuration for this.

Reply:
@jturner65 Does it work for other scenes?

Reply:
In this very early WIP i have it hardcoded to run in viewer, regardless of scene, so it should.

Reply:
just tested running viewer in other scenes.  Seems to work :)